share/nisar/defaults/focus.yaml

```
@@ -415,11 +415,18 @@ runconfig:
             # cal pulses.
             zero_fill_gaps: True
 
-            # If true, subtract the azimuth mean from the raw data, which is
-            # useful when the caltone is nearly constant in azimuth but not
-            # stationary in range.  Calculating the mean and subtracting it is
-            # done in blocks using rangecomp.block_size
-            nullify_azimuth_mean: True
+            caltone:
+                # If "azimuth_mean", subtract the azimuth mean from the raw
+                # data, which is useful when the caltone is nearly constant in
+                # azimuth but not stationary in range.  Calculating the mean
+                # and subtracting it is done in blocks using
+                # rangecomp.block_size
+                # If "wavelet" then subtract an estimate that varies in 2D.
+                # No correction when null.
+                # wavelet | azimuth_mean | null
+                algorithm: azimuth_mean
+                frequency: 1216.904e6
+                wavelet_size: 64
```

How about somethign like this? (Feel free to improve verbiage)

```suggestion
                # Length of constant-overlap add (COLA) window used
                # for wavelet caltone removal. Ignored if `algorithm` is not
                # 'wavelet'.
                wavelet_size: 64
```